client/recipient.go

```diff
@@ -65,7 +65,8 @@ type RecipientDetails struct {
 	WebhookName string `json:"webhook_name,omitempty"`
 	WebhookURL  string `json:"webhook_url,omitempty"`
 	// webhook only
-	WebhookSecret string `json:"webhook_secret,omitempty"`
+	WebhookSecret  string          `json:"webhook_secret,omitempty"`
+	WebhookHeaders []WebhookHeader `json:"webhook_headers"`
```

Comment:
I had omit empty set here since all the other ones do but it was causing an issue where len(0) when we want to unset all the headers wasn't getting sent and so the headers were always sticking around. lmk if there's a better way to do this in the TF schema instead

Reply:
I think most of the omitempty shouldn't be there, tbh. They're not harming anything because of the way the client is used by the provider but they're probably not strictly "correct" either